bundles/AdminBundle/Controller/Admin/AssetController.php

```diff
@@ -630,7 +630,7 @@ public function deleteAction(Request $request)
             $parentAsset = Asset::getById($request->get('id'));
 
             $list = new Asset\Listing();
-            $list->setCondition('path LIKE ?', [$parentAsset->getRealFullPath() . '/%']);
+            $list->setCondition('path LIKE ?', [Db::get()->escapeLike($parentAsset->getRealFullPath()) . '/%']);
```

Comment:
I'm just thinking, maybe we should provide `escapeLike()` on the abstract listing class as well, as it is used quite often and we should avoid the static calls? wdyt?